packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx

```diff
@@ -216,7 +216,7 @@ export default {
 } as Meta<StoryControlProps>;
 
 const BottomSheetTemplate: ComponentStory<typeof BottomSheetComponent> = ({ ...args }) => {
-  const [isOpen, setIsOpen] = React.useState(false);
+  const [isOpen, setIsOpen] = React.useState(true);
```

Comment:
Is it open by default now? It seems not related.

Reply:
it is not open by default but i made this change because otherwise snapshot me bottomsheet won't be captured as its closed and we'll get snapshot of only "Open" button. we do this for Modal as well

Reply:
Oh! Makes sense.

Reply:
Makes sense, although @kamleshchandnani is it possible to detect if we are running on chromatic snapshot testing and only then make it true? Because now whenever you open the BottomSheet ka docs this bottomsheet will open up and interrupt the consumer.

Reply:
I can yes. but made it same as Modal. can do it for both then

Reply:
done [here](https://github.com/razorpay/blade/pull/1542/commits/678b74bdf82bb668737c16207eb6ed0519621b86)